charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -7,11 +7,13 @@ maintainers:
   - name: bismarck
   - name: gianrubio
     email: gianrubio@gmail.com
+  - name: gkarthiks
```

Comment:
IMO it would be great to have extra help, but up to the existing maintainers of this chart.